orchagent/fdborch.cpp

```diff
@@ -529,23 +816,197 @@ bool FdbOrch::removeFdbEntry(const FdbEntry& entry)
     status = sai_fdb_api->remove_fdb_entry(&fdb_entry);
     if (status != SAI_STATUS_SUCCESS)
     {
-        SWSS_LOG_ERROR("Failed to remove FDB entry. mac=%s, bv_id=0x%" PRIx64,
+        SWSS_LOG_ERROR("FdbOrch RemoveFDBEntry: Failed to remove FDB entry. mac=%s, bv_id=0x%lx",
                        entry.mac.to_string().c_str(), entry.bv_id);
-        return true; //FIXME: it should be based on status. Some could be retried. some not
+        return false; //FIXME: it should be based on status. Some could be retried. some not
```

Comment:
So we want to retry?  What if the fdb is already removed, and we could end up with retry forever?

Reply:
yes, i will change this.

Reply:
done